extractor/extractor/result.py

```diff
@@ -72,3 +72,13 @@ def skip(self, n: int) -> Result:
 
     def take(self, n: int) -> Result:
         pass
+
+    def to_json(self) -> Dict[str, Any]:
+        """
+        Convert the result into JSON suitable for use as data for modules.
+
+        TODO: Think about additional metadata we might want
+        """
+        return {
+            "filings": [f.to_json() for f in self],
```

Comment:
https://github.com/code-for-montana/nonprofit-data/pull/43/files#diff-e68e2639de57d960287faf4715b35682R64

Probably missing something but how are the quotation marks being added around the values here ^^? I would have kinda expected needing to do a `json.dumps` here as well?

Reply:
Here we just build a dictionary that contains a single key, mapped to a list of other dictionaries, and dump it to JSON in the formatter (or wherever). It's basically the same as how our `toJson` methods in Dart return a `Map`.

Reply:
For posterity: the real JSON encoding happens in `formatter.py` which is added as a follow-up to this PR. In this PR the app `dump` is just what's used as proof of concept.